joblib/externals/loky/backend/__init__.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+
+from .context import get_context
+
+LOKY_PICKLER = os.environ.get("LOKY_PICKLER")
+
+if sys.version_info > (3, 4):
+
+    @staticmethod
```

Comment:
This will be done in the next version of `loky`. Do you want me to change it in the joblib repo?
I think it is better to not touch to externals files but I can do it if needed.